app/routes/register-your-interest/check-your-answers-and-register-your-interest.js

```diff
@@ -0,0 +1,70 @@
+const session = require('../../session')
+const sessionKeys = require('../../session/keys')
+const urlPrefix = require('../../config/index').urlPrefix
+const callChargesUri = require('../../config/index').callChargesUri
+const ruralPaymentsEmail = require('../../config/index').ruralPaymentsEmail
+
+const PATH = `${urlPrefix}/register-your-interest/check-your-answers-and-register-your-interest`
+
+module.exports = [
+  {
+    method: 'GET',
```

Comment:
When running locally i seemed to be directed to the login screen as this route was protected by auth - i added the following to resolve

`auth: false`

Reply:
good catch!